docs/latest/dev/releasing.md

```diff
@@ -13,34 +19,96 @@ Set environment variables for use in subsequent steps:
 ```shell
 export MAJOR_VERSION=0
 export MINOR_VERSION=3
+export RELEASE_CANDIDATE_NUMBER=1
 export GITHUB_REMOTE=origin
 ```
 
 1. Clone the repo, checkout the `main` branch, ensure it’s up-to-date, and your local branch is clean.
-2. Create a topic branch to create the release notes and release docs. Reference previous [release notes][] for additional details.
-3. Sign, commit, and push your changes to your fork and submit a [Pull Request][] to merge the changes listed below
-   into the `main` branch. Do not proceed until all your PRs have merged and the [Build and Test][build-and-test GitHub action] has completed for your final PR:
-
-   1. Add Release Announcement.
-   2. Add Release Versioned Documents.
-
-   ``` shell
-      make docs-release TAG=v${MAJOR_VERSION}.${MINOR_VERSION}.0
-   ```
-
-4. Create a new release branch from `main`. The release branch should be named
+2. Create a topic branch for adding the release notes. Refer to previous [release notes][] for additional details.
```

Comment:
in markdown you can use 
```
1. xxxx
1. xxxx
```
https://www.markdownguide.org/basic-syntax/#ordered-lists